packages/data-uri-to-buffer/src/index.ts

```diff
@@ -5,7 +5,21 @@ export interface ParsedDataURI {
 	buffer: ArrayBuffer;
 }
 
+declare const __useCustomDecodeInTests: boolean;
+
 function base64ToArrayBuffer(base64: string) {
+	// fast path for Node.js:
+	if (typeof Buffer === 'function' && typeof __useCustomDecodeInTests === 'undefined') {
```

Comment:
I have an alternative proposal: how about we utilize `package.json` [conditional `exports`](https://nodejs.org/api/packages.html#conditional-exports), such that Node.js has a specialized entrypoint?

Reply:
Done! Let me know what you think.